### PR TITLE
pfile-utils (new formula)

### PR DESCRIPTION
**Pull request by Marvin182**
pfile-utils is a a collection of utilities for manipulating PFiles, the ICSI feature file archive format. It was invented to store training data for speech recognition system and neural networks and is still used by several research teams in the field.

quicknet is a software suite for training simple neural networks and required by pfile-utils.

Also see original pull request: https://github.com/Homebrew/homebrew/pull/47975

### CODE REVIEW

File: pfile-utils.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class PfileUtils < Formula
+  desc "Collection of utilities for manipulating PFiles (ICSI feature file)"
+  homepage "http://www1.icsi.berkeley.edu/Speech/qn.html"
+
+  url "https://github.com/Marvin182/pfile-utilities/archive/v0.51.tar.gz"
```
Comment (dpo) [inline pfile-utils.rb]: Please remove blank line.

File: quicknet.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  test do
+    assert File.exist? "#{bin}/qncopy"
+    assert File.exist? "#{bin}/qncopywts"
+    assert File.exist? "#{bin}/qndo"
+    assert File.exist? "#{bin}/qnmultifwd"
+    assert File.exist? "#{bin}/qnmultitrn"
+    assert File.exist? "#{bin}/qnnorm"
+    assert File.exist? "#{bin}/qnsfwd"
+    assert File.exist? "#{lib}/libquicknet3.dylib"
```
Comment (dpo) [inline quicknet.rb]: Isn't there a software test that could be performed here?

**Comment by Marvin182** [comment]
Hi @dpo,
So here the situation:
Neither pfile-utils nor quicknet were written by me and in fact there hasn't been any active development for years. This is research, nobody has time to maintain old software and I know a project that still uses quicknet. But pfiles are still around, and will be for a while, and therefore I thought it would be nice to have the utilities as a homebrew formula. Just to save some colleagues the manual installation.
BUT I know nothing about quicknet or why it is a dependency for pfile-utils. And because the documentation on how to create a formula is very basic, creating this pull request cost me already much more time than I intended to spend on it.

And here is what is gonna happen:
I don't have the knowledge to write a software test and I don't think it is required. If you want to write one, feel free to ahead. By now I don't care if this is merged or not, but I will not spend any more time on it. And if you get annoyed by a blank line you will have to remove it yourself.

Sorry for being a bit harsh, but I had a bad day and I don't think this formula is worse spending more time on it.

**Comment by dpo** [comment]
I'm closing this then. We're all doing this in our "free" time. Thanks anyways.

**Comment by Marvin182** [comment]
lol, why not just merging it? I'm sure there are plenty of packages without software test...

**Pull request closed without merging**